src/app/asset-page/artstor-viewer/artstor-viewer.component.scss

```
@@ -83,7 +80,22 @@ $desktop: 992px;*/
         color: $color-white;
     }
 
-
+    .btn--icon {
+        border-radius: 0;
+    }
+    // When remove from comparison, or present icon is by itself and needs a full border-radius
+    .btn--icon.standalone-remove-icon, .btn--icon.standalone-present-icon {
+        border-radius: 4px;
+    }
+    pharos-tooltip:first-of-type .btn--icon {
```

Would it be more understandable later if you added e.g. `.asset-viewer__buttons__button` (yuck) to each tooltip so you can select like this?

```suggestion
   .asset-viewer__buttons__button:first-child .btn--icon {
       ...
   }
```

---

Not a bad idea!